osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -864,7 +863,7 @@ private void updateVisibleBeatmapCount()
         {
             // Intentionally not localised until we have proper support for this (see https://github.com/ppy/osu-framework/pull/4918
             // but also in this case we want support for formatting a number within a string).
-            FilterControl.InformationalText = $"{"match".ToQuantity(Carousel.CountDisplayed, "#,0")}";
+            FilterControl.InformationalText = Carousel.CountDisplayed != 1 ? $"{Carousel.CountDisplayed} matches" : $"{Carousel.CountDisplayed} match";
```

Comment:
I know that's mostly not the point, but how much do we care about the change in number formatting here? The previous spec of `#,0` would add thousands grouping, the new way will not group thousands.

Not that I particularly care either way, but given it was explicitly put there...

Reply:
Going to apply again based on discord conversation: https://discord.com/channels/188630481301012481/589331078574112768/1119688755394125864